Day-6-7-8-9/src/components/CardComponent.js

```diff
@@ -1,30 +1,33 @@
-import React from "react";
+import React, { useContext } from "react";
 import { Link } from "react-router-dom";
+import ThemeContext from "./ThemeContext";
 
 const CardComponent = ({ user, showLink = false, showAllRepos = false }) => {
+  const { theme } = useContext(ThemeContext);
```

Comment:
it seems like this just you added for your reference. not used it.

Reply:
will remove